lib/cookieman.js

```diff
@@ -114,10 +114,27 @@ function clearAll (name) {
   return cleared
 }
 
+/**
+ * gets the value of a cookie, or returns null
+ *
+ * @public
+ * @param {String} name The cookie name
+ * @returns {Null} or {Boolean} Returns null if the cookie is not found or the cookie's value
+ */
+function cookieValue (name) {
+  var val = null
+  var match = document.cookie.match(new RegExp(name + '=' + '([^;]+)'))
```

Comment:
if we go with regex's, you may want to escape name in this instance. Don't have the RFC memorized but I imagine there's valid characters for a cookie name which may be significant in a regex.